autoscaler/controllers/gateway/config/quickwit.go

```diff
@@ -0,0 +1,46 @@
+package config
+
+import (
+	odigosv1 "github.com/keyval-dev/odigos/api/odigos/v1alpha1"
+	commonconf "github.com/keyval-dev/odigos/autoscaler/controllers/common"
+	"github.com/keyval-dev/odigos/common"
+)
+
+const (
+	qwUrlKey = "QUICKWIT_URL"
+)
+
+type Quickwit struct{}
+
+func (e *Quickwit) DestType() common.DestinationType {
+	return common.QuickwitDestinationType
+}
+
+func (e *Quickwit) ModifyConfig(dest *odigosv1.Destination, currentConfig *commonconf.Config) {
+	if url, exists := dest.Spec.Data[qwUrlKey]; exists {
+		exporterName := "otlp/quickwit/" + dest.Name
+
+		currentConfig.Exporters[exporterName] = commonconf.GenericMap{
+			"endpoint": url,
+			"tls": commonconf.GenericMap{
+				"insecure": true,
+			},
+		}
+
+		if isTracingEnabled(dest) {
+			currentConfig.Service.Pipelines["traces/quickwit"] = commonconf.Pipeline{
+				Receivers:  []string{"otlp"},
+				Processors: []string{"batch"},
```

Comment:
Adding the `otlp` receiver and `batch` processor was moved out of each destination implementation and into the caller to be applied once for each destination.

Is your gateway collector able to start this way?

Reply:
Not sure to understand this feedback. What I can say is this configuration with an otel-collector is working fine with Quickwit:

```yaml
receivers:
  otlp:
    protocols:
      grpc:
      http:

exporters:
  otlp/quickwit:
    endpoint: "quickwit:7281"
    tls:
      insecure: true

processors:
  batch:

service:
  pipelines:
    traces:
      receivers: [otlp]
      processors: [batch]
      exporters: [otlp/quickwit]
    logs:
      receivers: [otlp]
      processors: [batch]
      exporters: [otlp/quickwit]
```

Is it answering your question? Thanks